README.md

```diff
@@ -297,6 +297,23 @@ Following outputs are available
 | `json`        | String  | JSON output of tags and labels                                                             |
 | `bake-file`   | File    | [Bake file definition](https://docs.docker.com/build/customize/bake/file-definition/) path |
 
+Alternatively, each output is also exported as an environment variable:
+
+* `DOCKER_METADATA_OUTPUT_VERSION`
+* `DOCKER_METADATA_OUTPUT_TAGS`
+* `DOCKER_METADATA_OUTPUT_LABELS`
+* `DOCKER_METADATA_OUTPUT_JSON`
+* `DOCKER_METADATA_OUTPUT_BAKE_FILE`
+
+So it can be used with our [Docker Build Push action](https://github.com/docker/build-push-action/):
+
+```yaml
+- uses: docker/build-push-action@v3
+  with:
+    build-args: |
+      DOCKER_METADATA_OUTPUT_JSON
+```
```

Comment:
I can't find docs in build-push-action that describe the syntax for this - we should probably add that on merging this.

Reply:
Yes it's a core implementation where you can use `--build-arg` flag without a value: https://docs.docker.com/engine/reference/commandline/build/#-set-build-time-variables---build-arg

> You may also use the `--build-arg` flag without a value, in which case the value
> from the local environment will be propagated into the Docker container being
> built:
> 
> ```console
> $ export HTTP_PROXY=http://10.20.30.2:1234
> $ docker build --build-arg HTTP_PROXY .
> ```